src/screens/HomeScreen/components/NameBarContainer.js

```diff
@@ -25,18 +24,27 @@ const handleClearButtonPress = async () => {
 const NameBarContainer = () => {
   const [name, setName] = useState('');
   const [inputText, setInputText] = useState('');
-  const [userData, setUserData] = useState('');
+  const [userData, setUserData] = useState({});
 
   useEffect(() => {
     getData().then((value) => setName(value));
```

Comment:
If you're just calling 1 function with the same arguments as the promise resolver method you could save a few characters by changing:

```js
getData().then((value) => setName(value))
```

to

```js
getData().then(setValue)
```

The same is true for the line below that starts with `getUserWordList(name).then(..)`.